src/config/playpenconfig.rs

```diff
@@ -0,0 +1,68 @@
+use std::path::{PathBuf, Path};
+
+use super::tomlconfig::TomlPlaypenConfig;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct PlaypenConfig {
+    editor: PathBuf,
```

Comment:
Turtles all the way down ;) I guess that we should not worry about it atm. book.js and index.hbs are not as parametrized and more coupled than one would like anyway.

This might be a good material for a refactor in another PR. What do you think @azerupi?